bot/cogs/belts.json

```
@@ -0,0 +1,63 @@
+{
+    "Branco": {
+        "objectives" : [
+            "- Finalizar um projeto sozinho",
+            "- Apresentar o teu projeto a outros ninjas",
+            "- Saber o nome de 5 ninjas e 2 mentores",
+            "- Completar o Lightbot"],
+        "color" : "FFFFFF"
+        },
+    "Amarelo" : {
+        "objectives" : [
+            "- Estar presente em 4 sessões", 
+            "- Fazer um projeto e apresentar à mesa"],
+        "color" : "F7E129"
+    },
+    "Azul" : {
+        "objectives" : [
+            "- Ajudar como mentor numa sessão",
+            "- Saber o nome de 7 ninjas e 4 mentores",
+            "- Conseguir completar o código que os mentores têm preparado para ti",
+            "- Fazer um site"],
+        "color" : "296DF7"
+        },
+    "Verde" : {
+        "objectives" : [
+            "- Apresentar um projeto para o Dojo inteiro",
+            "- Chegar a 5 Kyu em CodeWars",
+            "- Missão secreta nº2",
```

```suggestion
            "- Missão secreta n.º2",
```

Since "n" is an abbreviation of "número" it must have a dot.

---

I changed it in the most recent commit and also added the circle emoji to the private message as requested.